domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/DirectionsQuestionnaire.java

```diff
@@ -53,7 +56,8 @@ public DirectionsQuestionnaire(
         List<UnavailableDate> unavailableDates,
         YesNoOption expertRequired,
         YesNoOption permissionForExpert,
-        ExpertRequest expertRequest
+        ExpertRequest expertRequest,
```

Comment:
This could be refactored by adding a build builder and all args constructor to the top of the model class: 


**Ref:**
- https://github.com/hmcts/cmc-claim-store/blob/master/src/main/java/uk/gov/hmcts/cmc/claimstore/models/courtfinder/Address.java
- https://projectlombok.org/features/Builder   
- https://projectlombok.org/features/constructor